tests/unit_tests/tools/requests/test_tool.py

```diff
@@ -0,0 +1,100 @@
+import asyncio
+from typing import Any, Dict
+
+import pytest
+
+from langchain.requests import TextRequestsWrapper
+from langchain.tools.requests.tool import (
+    RequestsDeleteTool,
+    RequestsGetTool,
+    RequestsPatchTool,
+    RequestsPostTool,
+    RequestsPutTool,
+    _parse_input,
+)
+
+
+class _MockTextRequestsWrapper(TextRequestsWrapper):
```

Comment:
nit: This is not technically a mock, it's a Fake since you're stubbing the methods and not setting expectations